Tests/BasicAMRInterpolatorTest/AMRInterpolatorTest.cpp

```diff
@@ -56,53 +56,57 @@ int runInterpolatorTest(int argc, char *argv[])
     setupAMRObject(gr_amr, interpolator_test_level_fact);
 
     // Setup the AMRInterpolator
-    int num_points = 2;
-    pp.query("num_points", num_points);
+    int num_points = sim_params.num_points;
 
-    std::unique_ptr<double[]> chi_ptr{new double[num_points]};
-    std::unique_ptr<double[]> phi_ptr{new double[num_points]};
-    std::unique_ptr<double[]> interp_x{new double[num_points]};
-    std::unique_ptr<double[]> interp_y{new double[num_points]};
-    std::unique_ptr<double[]> interp_z{new double[num_points]};
+    double A_ptr[num_points];
+    double B_ptr[num_points];
+    double B_dx_ptr[num_points];
+    double interp_x[num_points];
+    double interp_y[num_points];
+    double interp_z[num_points];
```

Comment:
I don't feel too strongly about this but I prefer to avoid C-style arrays where possible and use `std::array`s or `std::vector`s instead. Could you replace these with `std::vector<double>`s?